requirements-pre-commit.txt

```diff
@@ -1,6 +1,7 @@
 pre-commit==2.15.0
 black==21.9b0
-sentry-flake8==2.0.0
+flake8==3.9.2
```

Comment:
Could bump flake8 to latest, changelog seems fine to me.

Reply:
planning to do that as a second PR, there's a few things that need changing to make it work heh